<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SidechainWithdrawalConfirmationDialog</class>
 <widget class="QDialog" name="SidechainWithdrawalConfirmationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Confirm Withdrawal</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="labelRefundDest">
     <property name="font">
      <font>
       <family>Noto Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>xyz</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <family>Noto Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Destination:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="labelMcFee">
     <property name="font">
      <font>
       <family>Noto Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="labelWTAmount">
     <property name="font">
      <font>
       <family>Noto Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="labelFee">
     <property name="font">
      <font>
       <family>Noto Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <family>Noto Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Refund Destination:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="labelDest">
     <property name="font">
      <font>
       <family>Noto Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>abc</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <family>Noto Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mainchain Fee:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Noto Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Withdrawal Amount:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Noto Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sidechain Fee:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Are you sure you want to create this withdrawal?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
